fixed grammar (#2336)
arbor-sim/arbor: doc/python/interconnectivity.rst

               # to the "syn" target label on cell gid with weight 0.01 and delay of 10 ms.
               source  = (2, "detector") # gid and locset label of the source
               target = "syn" # gid of the target is determined by the argument to `connections_on`.

>                w    = 0.01  # weight of the connection. Correspondes [corresponding?] to 0.01 μS on expsyn mechanisms

Reviewer: ```suggestion
               weight = 0.01  # unit/scaling depends on the synapse used; commonly chosen as μS 
```
Synapses get passed the `weight` given to `connection` as is when receiving an event. Thus,
each NMODL file may interpret the weight differently, but `uS` is a common (and sensible) choice.

>                d    = 10 * arbor.units.ms # delay

Reviewer: ```suggestion
               delay = 10 * arbor.units.ms # delay
```

>                return [arbor.connection(source, target, w, d)]

Reviewer: ```suggestion
               return [arbor.connection(source, target, weight, delay)]
```

               # to the "syn" target label on cell gid with weight 0.01 and delay of 10 ms.
               source  = (2, "detector") # gid and locset label of the source
               target = "syn" # gid of the target is determined by the argument to `connections_on`.
               weight = 0.01  # unit/scaling depends on the synapse used; commonly chosen as μS 

>                d    = 10 * U.ms # delay

Reviewer: ```suggestion
               d    = 10 * U.ms # delay
```
When we are writing delay anyhow, we can just call the variable that, no?
